src/components/item-filter/index.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState, useEffect } from 'react';
+import clsx from 'clsx';
+import NiceSelect from '@ui/nice-select';
+import { Range } from 'react-range';
+import { IRenderTrackParams } from 'react-range/lib/types';
+import { selectFilteredItems } from '../../features/leda-nft/store/leda-nft.slice';
+import useAppSelector from '../../store/hooks/useAppSelector';
+import SliderTrack from '../ui/input-range/slider-track';
+import SliderThumb from '../ui/input-range/slider-thumb';
+
+type Props = {
+  setNfts: any;
+};
+
+type PriceRangeType = {
+  from: number;
+  to: number;
+};
+
+type FilterType = {
+  likesDirection: string;
+  NFTauthor: string;
+  NFTtitle: string;
+  NFTdescription: string;
+  priceRange: PriceRangeType;
+};
+
+type TargetType = {
+  target: {
+    value: string;
+  };
+};
+
+type LikesHandleType = {
+  value: string;
+  text: string;
+  direction: string;
+};
+
+const ItemFilter = ({ setNfts }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [valuesRange, setValuesRange] = useState([0.1, 4]);
+
+  const [filterData, setFilterData] = useState({
+    likesDirection: '',
+    NFTauthor: 'all',
+    NFTtitle: 'all',
+    NFTdescription: 'all',
+    priceRange: {
+      from: 0.1,
+      to: 4,
```

Comment:
Add a TODO that says this values are just for testing and will be addressed dynamically down the road. Since having them hardcoded won't make sense since prices are dynamic. (Maybe we should define a business rule to put minimum prices and a max price as well